buildlib/azure-pipelines-release.yml

```diff
@@ -102,9 +102,23 @@ stages:
     condition: eq(dependencies.Prepare.outputs['CheckRelease.Result.Launch'], 'True')
     jobs:
       - template: az-distro-release.yml
-      - template: jucx/jucx-publish.yml
         parameters:
-          ${{ if eq(variables['Build.Reason'], 'IndividualCI') }}:
-            target: publish-release
-          ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
-            target: package
+          name: x86_64
+          demands: ucx_docker
+
+      - template: az-distro-release.yml
+        parameters:
+          name: aarch64
+          demands: ucx-arm64
+
+      - template: jucx/jucx-build.yml
+        parameters:
+          name: amd64   # x86_64
+          container: centos7_cuda11_x86_64
+          demands: ucx_docker
+
+      - template: jucx/jucx-build.yml
+        parameters:
+          name: aarch64 # ARM
+          container: centos8_cuda11_aarch64
+          demands: ucx-arm64
```

Comment:
@abellina is that OK that ARM minimal distro is centos8?

Reply:
I am not sure. Asking internally